pkg/sync/sync.go

```diff
@@ -184,7 +184,7 @@ func copyObject(src, dst object.ObjectStorage, obj object.Object) error {
 		in, e := src.Get(key, 0, -1)
 		if e != nil {
 			if _, err := src.Head(key); err != nil {
-				return nil
+				return err
```

Comment:
This is a feature rather than a bug. For example, a file is found and deleted from source later, it will be ignored rather than reported as failed.

Have you run into problem because of this behavior?